lib/index.js

```diff
@@ -53,12 +53,13 @@ class Rooftop {
       }
 
       if (transformFn === true) transformFn = transform
-      if (transformFn === false) transformFn = (x) => x
+      else if (transformFn === false) transformFn = (x) => x
 
       return this.client[name].get(options)
-        .then((p) => { return W.map(p, transformFn) })
-        .tap((res) => { m[name] = res })
-        .yield(m)
+        .then((p) => W.map(p, transformFn))
+        .then((v) => m[name] = v)
+        .then(() => m)
```

Comment:
Ohhh I see the issue:

https://github.com/cujojs/rest#change-log

So they are using standard ES6 promises now instead of when. Ok, well we can get when back with this:

```js
W.resolve(this.client[name].get(options))
   .then(...)
```